Lesson3.html

```diff
@@ -0,0 +1,26 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title> Lesson3 </title>
+    </head>
+    <body>
+        <p> How do you feel today? </p>
+        <button id="goodButton"> Good Slightly Smiling Face Emoji.png </button><button id="badButton"> Bad Very Sad Emoji.png</button>
+        <p id="answer"></p>
+        <script>
+            let goodButtonCmp = document.getElementById("goodButton");
+            let badButtonCmp = document.getElementById("badButton");
+            let answerCmp = document.getElementById("answer");
+            function goodButtonClick(event) {
+                answerCmp.textContent = "I feel really good today Very Sad Emoji.png";
```

Comment:
You can not reference an image with text content. To add emoji to a string just copy paste its text representation from some site: https://emojipedia.org/grinning-face/. Then you do not need to have jpg files. If you wish to add an image to a page, here is a link with some instructions: https://www.w3schools.com/html/html_images.asp